plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/ShellMatchingMatcherIsAvailable.java

```diff
@@ -39,13 +38,20 @@ public ShellMatchingMatcherIsAvailable(Matcher<?>... matchers) {
 		AndMatcher am  = new AndMatcher(matchers);
 		this.matcher = am;
 	}
+	
+	/**
+	 * Creates new ShellMatchingMatcherIsAvailable wait condition with specified shell text.
+	 * @param text shell text
+	 */
+	public ShellMatchingMatcherIsAvailable(String text){
```

Comment:
The above solution would mean moving ShellLookup to reddeer.swt which makes sense for me.